### PR TITLE
Feature small fixes

### PR DESCRIPTION
**Pull request by ricklamers**

### CODE REVIEW

File: services/jupyter-server/app/app/core/start_server.py [hunk 1]
```diff
@@ -4,6 +4,7 @@
 from typing import Any, Dict
 
 from jupyterlab.labapp import LabApp
+from flask import current_app
```
Comment (yannickperrenet) [inline services/jupyter-server/app/app/core/start_server.py:7]: I don't see why this addition is needed.

File: services/jupyter-server/app/app/core/start_server.py [hunk 2]
```diff
@@ -41,6 +42,7 @@ def _write_server_info_to_file(server_info: Dict[str, Any],
 
 
 def main():
+
```
Comment (yannickperrenet) [inline services/jupyter-server/app/app/core/start_server.py:45]: Also this addition does not add anything.

File: services/update-server/app/app/utils.py [hunk 3]
```diff
@@ -3,7 +3,7 @@
 
 def orchest_ctl(client, command):
 
-    return client.containers.run("orchestsoftware/orchest-ctl:latest", command, detach=True,
+    return client.containers.run("orchestsoftware/orchest-ctl:latest", command, name='orchest-ctl', detach=True, auto_remove=True,
```
Comment (yannickperrenet) [inline services/update-server/app/app/utils.py:6]: I see this function is repeated about three times in different services, should we move this into a module in our internal package?

**Review by yannickperrenet — CHANGES_REQUESTED** [review]
Ready to merge after the `start_server` fixes 👍

File: services/update-server/app/app/utils.py (hunk 3 above)
Comment (ricklamers) [reply services/update-server/app/app/utils.py:6]: It was two times, not three. And I was still not sure whether I wanted to put it in the 'lib' hence postponing for a bit. Aware of the duplication. Let's discuss the "what do we put in lib" question. I felt this was too specific to the web apps (update-server and orchest-webserver) to put in lib.

**Pull request merged**